manifests/init.pp

```diff
@@ -304,6 +340,20 @@
   $oauth_consumer_secret      = $foreman_proxy::params::oauth_consumer_secret
 ) inherits foreman_proxy::params {
 
+  # Port is deprecated
+  if $port {
+    warning("${::hostname}: foreman_proxy::port is deprecated; please use http_port or https_port instead")
+    $real_ssl        = $ssl
+    $real_http       = !$ssl
+    $real_http_port  = $port
+    $real_https_port = $port
```

Comment:
Setting both of these to $port is probably going to cause an issue - should we only set real_https_port if $ssl, and real_http_port if !$ssl?